src/execution_plan/ops/op_project.c

```diff
@@ -22,6 +22,14 @@ static AR_ExpNode** _getOrderExpressions(OpBase *op) {
     return _getOrderExpressions(op->parent);
 }
 
+int _actualAliasesCount(char** aliases){
```

Comment:
Missing comment, explaining what this function does and more importantly why.

Reply:
done